templates/emails/backer.monthlyreport.hbs

```diff
@@ -279,6 +280,8 @@ Subject: {{{capitalize month}}} Report
       <div class='Section Intro'>
         <h2>Hi {{recipient.firstName}}!</h2>
         <p>We are starting this new monthly email to keep you posted on the latest activities of the collectives that you are supporting. Let us know if you like this and do not hesitate if you have any feedback or ideas. Just reply to this email!</p>
+        <p>If you are in NYC, come join us for a drink 🍻 next week Thursday March 9 7pm at the Swift Hiberian Lounge. RSVP on our new events page 😊 <a href="https://opencollective.com/opencollective/events/meetup-1?{{utm}}">https://opencollective.com/opencollective/events/meetup-1</a></p>
```

Comment:
done.

I fixed the paid tier, see the PR I merged. Just need to deploy and test on prod.